miniapp/USERGUIDE.md

```diff
@@ -622,6 +622,44 @@ mini app scheme and should close external webview.
 
 Using `#ExternalResultHandler.emitResult(String)` to transmit the url string to mini app view.
 
+### File picking
+**API Docs:** [MiniAppFilePicker](api/com.rakuten.tech.mobile.miniapp.file/-mini-app-file-picker/)
+
+The mini app is able to pick a file which is requested using [input type="file"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file).
+HostApp can use `MiniAppFilePicker` object to pass through `MiniApp.create(appId: String, miniAppMessageBridge: MiniAppMessageBridge, miniAppFilePicker: MiniAppFilePicker)`.
+
+- Implement `MiniAppFilePicker` in Host App Activity.
+
+```kotlin
+var filePathHostCallback: ValueCallback<Array<Uri>>? = null
+val filePickingReqCode = REQUEST_CODE
+
+miniAppFilePicker = object : MiniAppFilePicker {
+    override fun requestFile(
+        filePathCallback: ValueCallback<Array<Uri>>?,
+        callback: (requestCode: Int) -> Unit
+    ) {
+        filePathHostCallback = filePathCallback
+        callback.invoke(filePickingReqCode)
```

Comment:
from what I understand is Hostapp can ignore this callback when they want to implement their own file request so you can document about it too.

Reply:
Yeah, I have wrote about it in the user guide.